.github/workflows/release.yml

```diff
@@ -57,13 +57,13 @@ jobs:
           file_name: .prod.env
           fail_on_empty: false
       - name: Test
-        run: npm run --workspaces test --if-present
+        run: yarn run test
       - name: Build
-        run: npm run --workspace packages build --if-present
```

Comment:
I switched from only build `packages` to building everything. It didn't seem right to skip `apps/`.